nncf/onnx/graph/metatypes/onnx_metatypes.py

```diff
@@ -506,7 +506,6 @@ class ONNXDeformableConvolutionMetatype(ONNXOpMetatype):
 OPERATIONS_WITH_BIAS_METATYPES = [
     ONNXConvolutionMetatype,
     ONNXDepthwiseConvolutionMetatype,
-    ONNXConvolutionTransposeMetatype,
```

Comment:
@kshpv, @KodiaqQ Does ONNX support Linear/MatMul operation in BC algorithm?

Reply:
@KodiaqQ is right